src/main/java/org/folio/circulation/services/PubSubRegistrationService.java

```diff
@@ -68,7 +77,6 @@ private PubSubRegistrationService() {
       logger.error("Module's publishers were not unregistered from PubSub.", exception);
     }
 
-    return allOf(list.toArray(new CompletableFuture[0])).thenApply(r -> true)
-      .whenComplete((r, e) -> client.close());
+    return allOf(list.toArray(new CompletableFuture[0])).thenApply(r -> true);
```

Comment:
The clause `.whenComplete((r, e) -> client.close());` is not needed anymore here?

Reply:
This was a workaround for a bug in mod-pubsub-client, which is now apparently fixed, since there is no method `close()` in the updated version. But it does not matter, since I reverted this upgrade. There is a new API in mod-pubsub 2.0 which allows us to unregister the module using one line of code. I will make use of it in a separate PR.